### PR TITLE
Add Foreman 2.0 and Katello 3.15

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: releasetool.yaml [hunk 1]
```diff
@@ -100,4 +108,15 @@ client:
...
         - sles12
         - fedora29
+    client-2.0:
+      version: '2.0'
+      full_version: '2.0.0'
+      operating_systems:
+        - rhel5
+        - rhel6
+        - rhel7
+        - rhel8
```
Comment (evgeni) [inline releasetool.yaml]:
```suggestion
        - el8
```

File: releasetool/data/roles/import_gpg_private/tasks/main.yaml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 ---
 - name: import gpg key
-  shell: "gopass show '{{ pass_name_key }}' | gpg2 --homedir '{{ work_dir_gpg }}' --import"
+  shell: "gopass show '{{ pass_name_key }}' | gpg --homedir '{{ work_dir_gpg }}' --import"
```
Comment (evgeni) [inline releasetool/data/roles/import_gpg_private/tasks/main.yaml]: why was that needed?

Comment (evgeni) [reply releasetool/data/roles/import_gpg_private/tasks/main.yaml]: ah, on recent fedoras, g2 is a symlink to g, not gnupg1 anymore, nice

Comment (ehelms) [reply releasetool/data/roles/import_gpg_private/tasks/main.yaml]: I had some issues with gpg2 and importing, will switch it back

File: README.md [hunk 3]
```diff
@@ -33,6 +33,16 @@ Each project can later define their own, project-specific, settings. The last le
...
 The general usage of the tool is to call `releasetool ACTION PROJECT-VERSION`. Executing actions against the unversioned project definitions is not supported.
 
+If at any step the system asks you for the password to a GPG key, use crtl+v to paste the secret that was copied automatically to the clipboard.
+
+### Import GPG Key
+
+Before running most actions the associated GPG key will need to be imported. Example:
+
+```
+releasetool import_gpg_key foreman-2.0
```
Comment (evgeni) [inline README.md]:
```suggestion
releasetool import-gpg-private foreman-2.0
```

**Review by evgeni — CHANGES_REQUESTED** [review]
The command is import-gpg-private, not import-gpg-key.

**Comment by ehelms** [comment]
Sorry, my brain is just translating what it's used to.

**Review by evgeni — CHANGES_REQUESTED** [review]
The client is missing the foreman version

**Comment by ehelms** [comment]
I don't follow what you mean.

**Comment by evgeni** [comment]
That's because i forgot we got rid of that!

**Review by evgeni — APPROVED** [review]
